src/routes/docs/components/badge.md

```diff
@@ -121,13 +121,14 @@ You can also use SVG icons inside the badge elements.
 ```svelte example class="flex flex-wrap gap-2" hideScript
 <script>
 	import { Badge } from 'flowbite-svelte';
+  import { ClockSolid } from 'flowbite-svelte-icons';
 </script>
 <Badge color="dark" border>
-  <svg aria-hidden="true" class="w-3 h-3 mr-1" fill="currentColor" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" d="M10 18a8 8 0 100-16 8 8 0 000 16zm1-12a1 1 0 10-2 0v4a1 1 0 00.293.707l2.828 2.829a1 1 0 101.415-1.415L11 9.586V6z" clip-rule="evenodd"></path></svg>
+  <ClockSolid class="text-gray-800 dark:text-gray-400 w-2.5 h-2.5 mr-1.5"/>
```

Comment:
Setting `text-gray-800 dark:text-gray-400` is redundant here as those are inherited from the surrounding badge.

Reply:
Thanks. Updated.